ui-src/src/environments/environment.prod.ts

```diff
@@ -1,3 +1,5 @@
 export const environment = {
   production: true,
+  postUrl: "http://localhost:9191/configs",
+  getUrl: "http://localhost:9191/configs/"
```

Comment:
You only need the one line here so it should be like:

apiBaseUrl: "http://localhost:9191/"

`${apiBaseUrl}/config`